src/node/util.ts

```diff
@@ -20,19 +22,23 @@ export const paths = getEnvPaths()
  * On MacOS this function gets the standard XDG directories instead of using the native macOS
```

Comment:
interesting, I didn't know this! makes sense though. so this means programs use `~/.config` and `~/.local` even on Mac?

Reply:
Yeah originally code-server used the other dirs but it was changed to align with command line tools like the ones @jsjoeio mentioned. I think it's a bit of a grey area because you can probably argue code-server is more like an app than a command line tool, but at least this way we can reference a single directory in the docs.